docs/user/interfaces/asyncqueryinterface.rst

```diff
@@ -35,6 +35,15 @@ The system relies on the default AWS credentials chain for making calls to the E
 *  ``applicationId``, ``executionRoleARN`` and ``region`` are required parameters.
 *  ``sparkSubmitParameter`` is an optional parameter. It can take the form ``--conf A=1 --conf B=2 ...``.
 
+Starting with Flint 0.1.1, users could use AWS CloudWatch as an external metrics sink while configuring their own metric sources. For example::
+
+   plugins.query.executionengine.spark.config:
+   '{  "applicationId":"xxxxx",
```

Comment:
Is this escaped string?

Reply:
It's actually confusing, let me replace with the console request.